components/lotties/cta.tsx

```diff
@@ -1,8 +1,8 @@
 import { useEffect, useState } from 'react'
-import Lottie from 'react-lottie-player/dist/LottiePlayerLight'
+import Lottie from 'react-lottie-player'
```

Comment:
Dropped the deep path as it breaks the type resolution.

This does mean it's not using the `...Light` import anymore, but as far as I can tell there is no easy way for it to use that import with the correct type definition. Can you declare a module for a nested import like this in TS?

Cc @Jedibezy @TheoBr @JacobMGEvans for thoughts.

Reply:
Wouldn't hurt lol

Reply:
[I'll update this if they merge it](https://github.com/mifi/react-lottie-player/pull/51)